mlir/include/mlir/Dialect/MIOpen/serializable.h

```diff
@@ -0,0 +1,115 @@
+//===------- sqlite_db.h - MLIR sqlite database client ----------===//
```

Comment:
should this interface be stored under Tuning directory?

Reply:
It gives more clarity to the matching code. Sure, will do.